### PR TITLE
Add history of commands window

### PR DESCRIPTION
**Pull request by tobiasz135**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/manager/HistoryOfUsedCommandsSubscriber.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.command.gui.HistoryOfUsedCommandsWindow;
+import edu.kis.powp.observer.Subscriber;
+
+public class HistoryOfUsedCommandsSubscriber implements Subscriber {
+    HistoryOfUsedCommandsWindow historyOfUsedCommandsWindow;
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/HistoryOfUsedCommandsSubscriber.java]: should be private

**Review by kowallus — CHANGES_REQUESTED** [review]
Please apply OBJECT-oriented approach, not STATIC-class-approach. Minimize static fields and methods. We try to learn using objects.

**Review by tobiasz135 — COMMENTED** [review]
Task issues fixed

**Review by kowallus — APPROVED** [review]

File: src/main/java/edu/kis/powp/jobs2d/command/manager/HistoryOfUsedCommandsManager.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+
+
+
+
+
+
+
+
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/manager/HistoryOfUsedCommandsManager.java:44]: interesting formatting

**Review by kowallus — APPROVED** [review]

**Pull request merged**